build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
Had a chat with [programmerq](https://github.com/programmerq) which tought me that if no `USER` is specified in the Dockerfile, `docker exec` runs with root permissions, although it's recommended/best practice to not run it with root - so I guess we also should add a `USER` directive to the Dockerfile. `docker exec` has a `-u` flag to use other users or even root to run it, if needed.